TRANSLATING.md

```diff
@@ -135,6 +141,16 @@ If you want to take your reviews to the next level, here are some more questions
 
 ## Translation Tips
 
+### Good and Bad Adaptation
+
+Not everything written in English will have a 1:1 translation, so it is not only allowed, but also encouraged to make adaptations when necessary. Here are our opinions on what is a good or bad adaptation:
+
+Things like removing or adding a comma, making a paragraph easier to read according to the language's norms, and other small changes are examples of **good adaptation**, as the overall meaning is not being lost.
+
+On the other hand, changing the page structure (e.g. moving headings, asides, etc.), the content itself (e.g. different examples, new paragraphs, etc.), and not faithfully translating the page are examples of **bad adaptation**, as they make it harder for new i18n contributors to jump in and review/update translations.
+
+Note that the Team Docs goes through a lot of very technical, intentional wording decisions to convey how things exactly work and why. When translations don't try to be faithful to the original version, it's possible that the original meaning is lost, especially when some key words are removed or changed for another with a different meaning.
```

Comment:
```suggestion
### How and How Much to adapt your translation

Not everything written in English will have a 1:1 translation, so it is not only allowed, but also encouraged to make language-specific adaptations to text when necessary. The two main goals of your translation should be:

1. To represent the English documents as closely as possible, remembering that they have undergone extensive review to achieve their current form.

2. To provide a natural, comfortable reading experience for a native speaker in the language.

Things like removing or adding a comma, making a paragraph easier to read according to the language's structure and norms, and other small changes are examples of **helpful adaptations**, where the overall meaning is not changed.

On the other hand, changing the page structure (e.g. moving headings, asides, etc.), the content itself (e.g. different examples, new paragraphs, etc.), and not faithfully translating the page are examples of **unhelpful adaptations**. These changes are more difficult for i18n contributors to review because they require significantly more effort to ensure that no meaning is lost or changed.

Please note that our English documents are written only after significant research, collaboration and negotiation by a team of developers and writers. Everything from page titles to code example file names has been discussed and agreed upon, so translations must represent the existing text without significant adaptation. There may be room for improvement! But, in our project, that happens only in the English documentation, and then the changes are represented in our translations.

If you find a section that is difficult to translate into your language, then that might mean that the English docs need some work! Please file a GitHub Issue and let us know. If you have some ideas for improvement, please use an Issue to make suggestions to the English documentation.
```